pkg/volume/csi/csi_client.go

```diff
@@ -113,14 +177,54 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 	accessibleTopology map[string]string,
 	err error) {
 	klog.V(4).Info(log("calling NodeGetInfo rpc"))
+	if c.nodeV1ClientCreator != nil {
+		return c.nodeGetInfoV1(ctx, c.nodeV1ClientCreator)
```

Comment:
does the creator have to be an argument? it's already a field in the csiDriverClient struct

Reply:
Good catch. Cleaning up.